iteration/iteration_test.go

```diff
@@ -212,3 +212,74 @@ func (test *TestIterationRepository) TestUpdateIteration() {
 	assert.Equal(t, changedStart, *updatedIteration.StartAt)
 	assert.Equal(t, changedEnd, *updatedIteration.EndAt)
 }
+
+func (test *TestIterationRepository) TestLoadChildren() {
+	t := test.T()
+	resource.Require(t, resource.Database)
+	newSpace := space.Space{
+		Name: "Space To Test Listing of Iteration Children",
```

Comment:
thanks , done in https://github.com/almighty/almighty-core/pull/1008/commits/0419cd3eb5432f9e7201f6d00ce71d133d412a4c#diff-ac105ac399c839c07e21bac7fbbb8651R223